recipe/meta.yaml

```
@@ -2,50 +2,78 @@
 {% set version = "1.10.0" %}
 
 package:
-  name: "{{ name }}"
-  version: "{{ version }}"
+  name: {{ name }}
+  version: {{ version }}
 
 source:
   url: https://ftp.gnu.org/gnu/gsasl/{{ name }}-{{ version }}.tar.gz
   sha256: f1b553384dedbd87478449775546a358d6f5140c15cccc8fb574136fdc77329f
 
 build:
-  number: 1
+  number: 2
   # This package is a dependency of libhdfs3, which is primarily used with HDFS on Linux.
-  # It's currently not available on s390x and there is no libgcrypt on s390x.
-  skip: True  # [win or s390x]
+  skip: true  # [win]
   run_exports:
     # unknown.  Leaving defaults.
     - {{ pin_subpackage('libgsasl') }}
 
 requirements:
   build:
+    - {{ stdlib('c') }}
     - {{ compiler('c') }}
-    - make  # [unix]
     - libtool  # [not win]
+    - make     # [unix]
   host:
-    - krb5
-    - libgcrypt
-    - libgpg-error  # [linux]
-    - libntlm
+    - libkrb5 {{ krb5 }}
+    - libgcrypt {{ libgcrypt }}
+    - libgpg-error {{ libgpg_error }}  # [linux]
+    - libntlm {{ libntlm }}
 
 test:
   commands:
+    # Header files
     - test -f $PREFIX/include/gsasl-compat.h
     - test -f $PREFIX/include/gsasl-mech.h
     - test -f $PREFIX/include/gsasl.h
+    # Library files
     - test -f $PREFIX/lib/libgsasl.a
+    - test -f $PREFIX/lib/libgsasl.7.dylib    # [osx]
+    - test -f $PREFIX/lib/libgsasl.dylib      # [osx]
     - test -f $PREFIX/lib/libgsasl.so         # [linux]
     - test -f $PREFIX/lib/libgsasl.so.7       # [linux]
-    - test -f $PREFIX/lib/libgsasl.so.7.10.0   # [linux]
+    - test -f $PREFIX/lib/libgsasl.so.7.10.0  # [linux]
     - test -f $PREFIX/lib/libgsasl$SHLIB_EXT  # [unix]
-    - test -f $PREFIX/lib/libgsasl.7.dylib    # [osx]
+    # pkgconfig file
+    - test -f $PREFIX/lib/pkgconfig/libgsasl.pc
+    # Locale files (Linux only)
+    - test -f $PREFIX/share/locale/da/LC_MESSAGES/libgsasl.mo    # [linux]
```

Maybe we can do this?
```suggestion
    - test -f $PREFIX/share/locale/*/LC_MESSAGES/libgsasl.mo    # [linux]
```

---

conda build expands it into:
```
+ test -f /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/da/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/de/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/en@boldquot/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/en@quot/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/eo/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/es/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/fi/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/fr/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/ga/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/hu/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/id/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/it/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/nl/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/pl/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/pt_BR/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/ro/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/sk/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/sr/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/sv/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/uk/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/vi/LC_MESSAGES/libgsasl.mo /croot/libgsasl_1753858640990/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/share/locale/zh_CN/LC_MESSAGES/libgsasl.mo
09:59:07
INFO
build
/croot/libgsasl_1753858640990/test_tmp/run_test.sh: line 16: test: too many arguments
```

---

I replaced it with testing files in `for loop`